src/macaron/dependency_analyzer/cyclonedx.py

```diff
@@ -147,6 +149,8 @@ def collect_dependencies(self, dir_path: str, target_component: Component) -> di
             Local path to the target repo.
         target_component: Component
             The analyzed target software component.
+        recursive: bool
+            Set to False to get the direct dependencies only (default).
```

Comment:
Both `collect_dependencies` and `resolve_dependencies` have a `recursive` parameter that does the same thing, but for one the comment only says what happens when it is False, and the other only says what happens when it is True:
```
Set to False to get the direct dependencies only (default).
```
```
If True, perform transitive dependency resolution. Default: False.
```
For consistency, they should have the same description, which should ideally describe both behaviours, e.g.
```
Whether to get all transitive dependencies, otherwise only the direct dependencies will be returned (default: False).
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/840/commits/70ed39dcd26459b0fbc6d9e76fa9b87e8fdd53f5